app/assets/javascript/pageflow/linkmap_page/editor/models/area.js

```diff
@@ -119,5 +120,17 @@ pageflow.linkmapPage.Area = Backbone.Model.extend({
 
   remove: function() {
     this.collection.remove(this);
+  },
+
+  get: function(attr) {
+    if (attr === 'mask_perma_id') {
+      var maskId = this.attributes['color_map_component_id'];
+      if(maskId === undefined) {
+        maskId = this.attributes['mask_perma_id'];
+      }
+      return maskId;
+    } else {
+      return this.attributes[attr];
+    }
```

Comment:
I think it would be clearer to do this the other way around. 

`color_map_component_id` is the name that will be used in the future. `mask_perma_id` is something we support for legacy reasons. But now there is still a mix of old and new names across the code base: `EditAreaView` uses the new name, other callers of `Area.get` still use the old name.

Instead I'd suggest only using the new name and having this method as the only place where `mask_perma_id` is still mentioned as a fallback.